lib/inc/hocon/config_exception.hpp

```diff
@@ -12,10 +16,10 @@ namespace hocon {
      */
     struct config_exception : public std::runtime_error {
         config_exception(config_origin const& origin, std::string const& message) :
-                runtime_error(origin.description() + ": " + message) { }
+                runtime_error(_("{1}: {2}", origin.description(), message)) { }
```

Comment:
Would it make more sense to translate `origin.description()` and `message`? Or are those already translated?